torchvision/transforms/functional.py

```diff
@@ -360,7 +360,9 @@ def normalize(tensor: Tensor, mean: List[float], std: List[float], inplace: bool
     return F_t.normalize(tensor, mean=mean, std=std, inplace=inplace)
 
 
-def _compute_output_size(image_size: Tuple[int, int], size: List[int], max_size: Optional[int] = None) -> List[int]:
+def _compute_resized_output_size(
```

Comment:
@datumbox please check this renaming to solve colliding function names and comment out if you approve or disagree. Thanks!